components/inputgroup/index.css

```diff
@@ -10,11 +10,453 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+    }
+  }
+
+  &:active {
+    background-color: var(--mod-combobox-background-color-active, var(--spectrum-combobox-background-color-active));
+    border-color: var(--mod-combobox-border-color-active, var(--spectrum-combobox-border-color-active));
+
+    &::placeholder {
+      color: var(--mod-combobox-font-color-active, var(--spectrum-combobox-font-color-active));
+    }
+  }
+
+  &:focus-visible,
+  &:focus-ring,
+  &.is-focused {
+    background-color: var(--mod-combobox-background-color-key-focus, var(--spectrum-combobox-background-color-key-focus));
+    border-color: var(--mod-alias-infieldbutton-fill-border-color-key-focus, var(--spectrum-alias-infieldbutton-fill-border-color-key-focus));
+
+    &::placeholder {
+      color: var(--mod-combobox-font-color-key-focus, var(--spectrum-combobox-font-color-key-focus));
+    }
+  }
+}
+
+.spectrum-InputGroup--quiet {
+  border-radius: var(--mod-combobox-quiet-fieldbutton-border-radius, var(--spectrum-combobox-quiet-fieldbutton-border-radius));
+
+  .spectrum-InputGroup-textfield {
+    &:focus-within::after {
+      pointer-events: none;
+      content: '';
+      position: absolute;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      top: 0;
+      margin: calc(
+        (var(--mod-combobox-focus-indicator-gap, var(--spectrum-combobox-focus-indicator-gap)) +
+         var(--mod-combobox-border-width, var(--spectrum-combobox-border-width)) *
+         -1));
+      inline-size: calc(
+        100% +
+        var(--mod-combobox-border-width, var(--spectrum-combobox-border-width))
+      );
+      block-size: calc(
+        100%  +
+        (var(--mod-combobox-focus-indicator-gap, var(--spectrum-combobox-focus-indicator-gap)) * 2) +
+        (var(--mod-combobox-border-width, var(--spectrum-combobox-border-width)) *
+         2)
+      );
+      margin: auto;
+      border: 0;
+      border-bottom-style: solid;
+      border-bottom-width: var(--mod-combobox-focus-indicator-thickness, var(--spectrum-combobox-focus-indicator-thickness));
+      border-bottom-color: var(--mod-combobox-focus-indicator-color, var(--spectrum-combobox-focus-indicator-color));
+      border-radius: 0;
+    }
+  }
+
+  .spectrum-InputGroup-button {
+    inline-size: auto;
+
+    .spectrum-PickerButton-fill {
+      border: none;
+    }
+  }
+
+  .spectrum-InputGroup-input {
+    padding-block-end: var(--mod-combobox-spacing-block-start-to-border, var(--spectrum-combobox-spacing-block-start-to-border));
+    padding-inline-start: var(--mod-combobox-spacing-inline-start-to-textfield, var(--spectrum-combobox-spacing-inline-start-to-textfield));
+    background-color: var(--mod-combobox-background-color, var(--spectrum-combobox-background-color));
+  }
+
+  .spectrum-InputGroup-input:disabled {
+    background-color: transparent;
+    border-bottom-color: var(--mod-combobox-border-color-disabled, var(--spectrum-combobox-border-color-disabled));
+  }
+}
+
+/* Date Picker*/
```

Comment:
The rest of this file contains styles for Date Picker with placeholder tokens, however, some of classes used overlap with Combobox.